### DISCUSSION TITLE
TF1 and M6 channels 720->480p

### DISCUSSION THREAD

**Discussion by Alchimisto** [iptv-org/community#212] [Q&A] [answered]
Hello, Is it possible to have theses channels in 480p 
In my experience, there's channels in 720p who are stable, but these are not, I don't know if there's a clue to correct that...
Thanks.

**Comment by dct-infra** [comment, marked as answer]
@Alchimisto Even in 480p it's quite difficult to find free/legit access to those 2 channels.
If you have 480p working links feel free to add them, meanwhile, if we find a way to have more stable streams we'll add them right away.

**Reply by Alchimisto** [reply]
I see now, thank you 🙏
